frontend/src/metabase-lib/metadata/Schema.ts

```diff
@@ -1,20 +1,30 @@
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-nocheck
-import { titleize, humanize } from "metabase/lib/formatting";
-import Base from "./Base";
+import { humanize, titleize } from "metabase/lib/formatting";
+import { NormalizedSchema } from "metabase-types/api";
 import type Metadata from "./Metadata";
 import type Database from "./Database";
 import type Table from "./Table";
-/**
- * Wrapper class for a {@link Database} schema. Contains {@link Table}s.
- */
 
-export default class Schema extends Base {
-  id: string;
-  name: string;
-  database: Database;
-  tables: Table[];
-  metadata: Metadata;
+export default class Schema {
+  private readonly schema: NormalizedSchema;
+  metadata?: Metadata;
+  database?: Database;
+  tables: Table[] = [];
```

Comment:
`tables` would be always set to an array during hydration, even if there are no tables in the store. Let's set this to an empty array to be typescript complaint.